### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by YuriyBilskiy**
[DEMO LINK](https://yuriybilskiy.github.io/react_sum/)

### CODE REVIEW

**Comment by YuriyBilskiy** [comment]
> https://yuriybilskiy.github.io/react_sum/



> https://yuriybilskiy.github.io/react_sum/

File: src/App.jsx [hunk 1]
```diff
@@ -1,14 +1,25 @@
...
+      />
+      <Sum
+        a={10}
+      />
+      <Sum
+        b={5}
+      />
+      <Sum />
+    </>
+  </div>
```
Comment (VitaliyBodnar) [inline src/App.jsx:24]: empty tag '<>' is used for making wrapper for the few tags, in this case, you added <div> and <> it's too much
so <div> wrapper is redundant

**Review by VitaliyBodnar — CHANGES_REQUESTED** [review]
Almost done!

**Review by VitaliyBodnar — CHANGES_REQUESTED** [review]
.

**Review by SanyaBratashchuk — APPROVED** [review]

**Pull request closed without merging**